os/logm/logm_process.c

```diff
@@ -81,7 +81,7 @@ int logm_task(int argc, char *argv[])
 			if (LOGM_STATUS(LOGM_BUFFER_OVERFLOW)) {
 				LOGM_STATUS_CLEAR(LOGM_BUFFER_OVERFLOW);
 			}
-			if (g_logm_overflow_offset >= 0 && g_logm_overflow_offset == g_logm_head) {
+			if (g_logm_overflow_offset >= 0 && g_logm_overflow_offset == g_logm_head && g_logm_dropmsg_count > 0) {
 				fprintf(stdout, "\n[LOGM BUFFER OVERFLOW] %d messages are dropped\n", g_logm_dropmsg_count);
```

Comment:
When buffer overflow happens, g_logm_dropmsg_count is set as 1 and grows up while overflow.
and g_logm_overflow_offset indicates where overflow happened at the same time.
so when g_logm_overflow_offset is greater than or equal to zero, g_logm_dropmsg_count is always greater than 0

Reply:
@EunBongSong Possible to get the message you said above. But fix is not correct. jeongarmy will provide a proper fix.

Reply:
Right, that conditions are enough but I missed to assign -1 to g_logm_overflow_offset at the first time.
So it is needed to initialize g_logm_overflow_offset as -1 in logm.c to resolve it instead.

Reply:
OK. 
Could you share patch that can resolve this issue ?

Reply:
Please refer commit 5010f28.